packages/engine/Source/Scene/Model/Extensions/Gpm/CorrelationGroup.js

```diff
@@ -0,0 +1,67 @@
+import Check from "../../../../Core/Check.js";
+
+/**
+ * Metadata identifying parameters using same correlation modeling and
+ * associated correlation parameters.
+ *
+ * @private
+ * @experimental This feature is not final and is subject to change without Cesium's standard deprecation policy.
+ */
+function CorrelationGroup(options) {
+  //>>includeStart('debug', pragmas.debug);
+  Check.typeOf.object("options.groupFlags", options.groupFlags);
+  Check.typeOf.object("options.rotationThetas", options.rotationThetas);
+  Check.typeOf.object("options.params", options.params);
+  //>>includeEnd('debug');
+
+  this._groupFlags = options.groupFlags;
+  this._rotationThetas = options.rotationThetas;
+  this._params = options.params;
+}
+
+Object.defineProperties(CorrelationGroup.prototype, {
+  /**
+   * Array of 3 booleans indicating if parameters delta-x delta-y delta-z
+   * used in the correlation group
+   *
+   * @memberof CorrelationGroup.prototype
+   * @type {boolean[]}
+   * @readonly
+   * @private
+   */
+  groupFlags: {
+    get: function () {
+      return this._groupFlags;
+    },
+  },
+
+  /**
+   * Rotations in milliradians about X, Y, Z axes, respectively
```

Comment:
This is taken from the JSON schema.

Since this is not only a _comment_, but actually refers to the _value of the property_, it's a bit different. One _could_ make a case of converting this to radians. It does raise the question about other forms of modifications. Do you think that this should be changed (possibly causing surprises for people who know "correlation groups in GPM")? 

We could also change its name to `kiloRotationThetas`. (That's not a lame joke, even though it may sound like one...)

Reply:
Given the intention of staying consistent with the existing extension, I think this is fine as is for now given the units are documented.